src/resources/js/events-admin.js

```diff
@@ -217,41 +217,41 @@ jQuery( document ).ready( function( $ ) {
 			$end_date = $( '#EventEndDate' );
 
 		var datepickerOpts = {
-			dateFormat     : date_format,
-			showAnim       : 'fadeIn',
-			changeMonth    : true,
-			changeYear     : true,
-			numberOfMonths : get_datepicker_num_months(),
-			firstDay       : startofweek,
-			showButtonPanel: true,
-			beforeShow     : function( element, object ) {
-				object.input.datepicker( 'option', 'numberOfMonths', get_datepicker_num_months() );
-				object.input.data( 'prevDate', object.input.datepicker( "getDate" ) );
-			},
-			onSelect       : function( selectedDate ) {
-				var option = this.id == "EventStartDate" ? "minDate" : "maxDate",
-					instance = $( this ).data( "datepicker" ),
-					date = $.datepicker.parseDate( instance.settings.dateFormat || $.datepicker._defaults.dateFormat, selectedDate, instance.settings );
-
-				if ( this.id === "EventStartDate" && $recurrence_type.val() !== 'None' ) {
-
-					var startDate = $( '#EventStartDate' ).data( 'prevDate' ),
-						dateDif = null == startDate ? 0 : date_diff_in_days( startDate, $end_date.datepicker( 'getDate' ) ),
-						endDate = new Date( date.setDate( date.getDate() + dateDif ) );
-
-					$end_date
-						.datepicker( "option", option, endDate )
-						.datepicker( "setDate", endDate );
+				dateFormat     : date_format,
```

Comment:
The end of an object should end where the beginning starts (if that makes sense).

How it is currently:
```js
var datepickerOpts = {
		dateFormat     : date_format,
		showAnim       : 'fadeIn',
		changeMonth    : true,
		changeYear     : true,
		numberOfMonths : get_datepicker_num_months(),
		firstDay       : startofweek,
		showButtonPanel: true,
		beforeShow     : function( element, object ) {
			object.input.datepicker( 'option', 'numberOfMonths', get_datepicker_num_months() );
			object.input.data( 'prevDate', object.input.datepicker( "getDate" ) );
		},
		onSelect       : function( selectedDate ) {
			var option = this.id == "EventStartDate" ? "minDate" : "maxDate",
				instance = $( this ).data( "datepicker" ),
				date = $.datepicker.parseDate( instance.settings.dateFormat || $.datepicker._defaults.dateFormat, selectedDate, instance.settings );

			if ( this.id === "EventStartDate" && $recurrence_type.val() !== 'None' ) {

				var startDate = $( '#EventStartDate' ).data( 'prevDate' ),
					dateDif = null == startDate ? 0 : date_diff_in_days( startDate, $end_date.datepicker( 'getDate' ) ),
					endDate = new Date( date.setDate( date.getDate() + dateDif ) );

				$end_date
					.datepicker( "option", option, endDate )
					.datepicker( "setDate", endDate );

			}
			else {
				dates
					.not( this )
					.not( '#recurrence_end' )
					.datepicker( "option", option, date );
			}
		}
	};
```

Tabbed in 1 deep:

```js
var datepickerOpts = {
	dateFormat     : date_format,
	showAnim       : 'fadeIn',
	changeMonth    : true,
	changeYear     : true,
	numberOfMonths : get_datepicker_num_months(),
	firstDay       : startofweek,
	showButtonPanel: true,
	beforeShow     : function( element, object ) {
		object.input.datepicker( 'option', 'numberOfMonths', get_datepicker_num_months() );
		object.input.data( 'prevDate', object.input.datepicker( "getDate" ) );
	},
	onSelect       : function( selectedDate ) {
		var option = this.id == "EventStartDate" ? "minDate" : "maxDate",
			instance = $( this ).data( "datepicker" ),
			date = $.datepicker.parseDate( instance.settings.dateFormat || $.datepicker._defaults.dateFormat, selectedDate, instance.settings );

		if ( this.id === "EventStartDate" && $recurrence_type.val() !== 'None' ) {

			var startDate = $( '#EventStartDate' ).data( 'prevDate' ),
				dateDif = null == startDate ? 0 : date_diff_in_days( startDate, $end_date.datepicker( 'getDate' ) ),
				endDate = new Date( date.setDate( date.getDate() + dateDif ) );

			$end_date
				.datepicker( "option", option, endDate )
				.datepicker( "setDate", endDate );

		}
		else {
			dates
				.not( this )
				.not( '#recurrence_end' )
				.datepicker( "option", option, date );
		}
	}
};
```

The example you gave is interesting because it uses comma-separated vars (which I'm no fan of) and aligns the closing brace with the beginning of a variable where the `var` is absent. Doing so is possible if you start using tabs after the `var` - which is kind of odd.

I hold that a clean way to do your example is as follows:

```js
var $recurrence_type = $( '[name="recurrence[type]"]' );
var $end_date = $( '#EventEndDate' );
var datepickerOpts = {
    variables: true
};
```

The benefits are:

1. Object ends are even with their beginning
1. The same amount of characters are used when writing that block of code (instead of spaces and a tab, you use `var` and a space - instead of a comma you use a semicolon)
1. The block of code is easier to manipulate when you want to add more (or remove) declarations. You don't need to convert semicolons to commas (and vice versa).
1. Diffs in version control will result in fewer lines changed when adding/removing variables.
1. There's [less typing](http://benalman.com/news/2012/05/multiple-var-statements-javascript/) when making an addition!